cue.yaml

```diff
@@ -34,7 +34,7 @@ pipeline:
       modroot: .
       packages: ./cmd/cue
       output: cue
-      ldflags: -w -buildid= -X cuelang.org/go/cmd/cue/cmd.version=${{package.version}}
+      ldflags: -buildid= -X cuelang.org/go/cmd/cue/cmd.version=${{package.version}}
```

Comment:
i thought passing in empty buildid= destroys information no? i would rather not pass one in, or have something meaningful like sha1 commit. but i can't remember the expected buildid format.